.github/workflows/autotag.yaml

```diff
@@ -0,0 +1,15 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
+jobs:
```

Comment:
This is a great idea but I'm not sure if we should automate it. Often you develop side-by-side in the Cosmo and engine branch. After merging and releasing the engine changes, you want to bump the version in the same cosmo branch. Additionally, what will happen when you do two releases in a row? Will it update an existing PR in cosmo?

In my opinion we should first automate version tagging and releasing before we running this experiment.